install_scripts/templates/kubernetes/node-join.sh

```diff
@@ -412,6 +414,14 @@ purgeNative
 
 if [ "$MASTER" -eq "1" ]; then
     exportKubeconfig
+
+    installCliFile \
+        "kubectl exec -c replicated" \
+        '$(kubectl get pods -o=jsonpath="{.items[0].metadata.name}" -l tier=master) --'
+    logSuccess "Installed replicated cli executable"
+
+    installAliasFile
```

Comment:
added another issue for this in 2.35
https://app.clubhouse.io/replicated/story/19400/replicated-alias-should-work-on-all-masters